samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,527 @@
+const
+    data = JSON.parse(document.getElementById('data').innerHTML),
+    scoreData = data[3],
+    colors = Highcharts.getOptions().colors.map(Highcharts.Color.parse),
+
+    // Defining recurring values.
+    monthExtremes = { min: 0, max: 26 },
+    weekExtremes = { min: 1, max: 5 },
+    paneOpeningAngles = { startAngle: 40.5, endAngle: 319.5 },
+    noLabelProp = { labels: { enabled: false } },
+    specialSeriesProps = {
+        showInLegend: false,
+        groupPadding: 0,
+        pointPadding: 0
+    },
+
+    // A gradient background for the inner circle (aka pane.)
+    toggleableGradient = {
+        pattern: undefined,
+        radialGradient: [1, 0.25, 0.1],
+        stops: [
+            [0, '#1f1836'],
+            [1, '#45445d']
+        ]
+    },
+
+    // A function which (re)sets the inner circles background to our gradient.
+    setGradient = function () {
+        const chart = this.series.chart;
+        chart.setMidPaneBg({
+            backgroundColor: toggleableGradient,
+            outerRadius: '75%'
+        });
+        chart.subtitle.element.style.opacity = 1;
+    },
+
+    // A function used in the creation of our second custom tooltip.
+    asColFieldStr = str => (
+        '<span class="col-display-fieldwrap">' +
+        '<span class="symbolSize" ' +
+        'style="color:{point.color};">●</span> ' + str + '</span>'
+    ),
+
+    // We create our teams, 1 serie per team.
+    teamNames = ['Ulambaator', 'Sofia', 'Asmara'],
+    teamColors = [
+        colors[9].tweenTo(colors[0], 0.25),
+        colors[9].tweenTo(colors[8], 0.65),
+        colors[9].tweenTo(colors[3], 0.85)
```

Comment:
`colors[9]` fails with an error for themes "Brand Light" and "Brand Dark" as there are only 8 colors for each theme.

For getting the colors array, something like this works:
`[...Highcharts.getOptions().colors, ...(Highcharts.getOptions().colors.length < 9 ? new Array(9 - Highcharts.getOptions().colors.length).fill('#bada55') : [])]` (and then map it with `parse` as in the 4th line) but readability could be improved and the place holder color picked so the chart looks good.

Reply:
I'd prefer that we just add more colors to those themes.

Reply:
@TorsteinHonsi Should this be resolved in this PR or will it be handled elsewhere?

Reply:
Doing it in this PR is fine by me.

Reply:
@KacperMadej On second thought, I don't think we should change the theme. Chances are there are themes out there with any number of colors, so our demo should support that. We use the rest operator as safeguard, something like this:

`colors[9 % colors.length].tweenTo(n)`